Platform/CommonBoardPkg/CfgData/Template_BootOption.dsc

```diff
@@ -21,7 +21,7 @@
 
     # !BSF NAME:{Boot Flags}
     # !BSF TYPE:{Combo}
-    # !BSF OPTION:{0:Normal, 1:A/B support, 2:Crash OS, 4:Trusty support, 5:Trusty and A/B support}
+    # !BSF OPTION:{0:Normal, 1:A/B support, 2:Crash OS, 4:Trusty support, 5:Trusty and A/B support, 16:Mender support}
     # !BSF HELP:{Specify boot flags (options)}
```

Comment:
is it a typo for "16:" , should it be "6"?

Reply:
No, 16 is correct because the boot flags map to bit-fields based on the macros defined here:

https://github.com/slimbootloader/slimbootloader/blob/610f66c19fc420e08e0581ceb6dbb313b8bbb9ba/BootloaderCommonPkg/Include/Guid/OsBootOptionGuid.h#L22

That is why "5" is Trusty (BIT2) and A/B support (BIT0) (BIT2 + BIT0 ==> 0x4 + 0x1). Since Medner is BIT4 it should be 16 is my understanding and based on my testing.

Reply:
Got it. I had thought we have already listed 0 ~ 5 here. so it is not bit wise value.